include/range/v3/view/drop.hpp

```diff
@@ -157,7 +157,7 @@ namespace ranges
                 static iterator_range<iterator_t<Rng>, sentinel_t<Rng>>
                 invoke_(Rng && rng, range_difference_type_t<Rng> n, concepts::RandomAccessRange*)
                 {
-                    return {next(begin(rng), n), end(rng)};
+                    return {next(begin(rng), n, end(rng)), end(rng)};
```

Comment:
The requirements are insufficient to guarantee that `next(begin(rng), n, end(rng))` is O(1). O(1) would additionally require `SizedRange` and (I think) `Assignable<iterator_t<Rng>&, sentinel_t<Rng>>`.